data/org.ldelossa.way-shell.gschema.xml

```diff
@@ -139,4 +139,18 @@
         </key>
     </schema>
 
+    <!--clock related settings-->
+    <schema path="/org/ldelossa/way-shell/clock/" id="org.ldelossa.way-shell.clock">
+        <key name="date-time-format" type="s">
+            <default>"%b %d %I:%M"</default>
+            <summary>The format to display the current time in on the panel.</summary>
+            <description>
+            This is passed as the `format` argument to g_date_time_format when rendering the clock
```

Comment:
Hey Rose. 

Can you make a note about changing this requiring a restart of Way-Shell?